### ISSUE TITLE
Make game card menu scrollable 

### ISSUE THREAD

**Issue by nyghtstalker** [HyperPlay-Gaming/hyperplay-desktop-client#531]
### Problem description

The game card menu can only hold six lines. After six, the seventh and further lines get cut off. This menu should be scrollable to help counter this issue.

### Feature description

Making the menu scrollable will allow for more content to be added without cut off.

### Additional information

![image](https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/assets/82125535/a661ea83-b892-4a17-bcf8-9d02b4097138)

**Comment by iamdavid0091** [comment]
Let's re-explore what elements actually need to be in this "dropdown".

**Comment by biliesilva** [comment]
I agree that we need to keep just 5 or 6 items here. so it will be very populated for a small component.

**Comment by iamdavid0091** [comment]
@biliesilva let's rethink what options makes sense in this "dropdown" as it relates to the actions that are available on the game details page/game card.

For example, the "Favorite" option is not needed due to the fact that we have the heart icon on the game card. Another example: game settings probably doesn't need to live in this "dropdown".

**Comment by biliesilva** [comment]
Yes, both **_Favorite_** and **_Play_** already have an icon for it on the same card. We can keep these options: 
- **Details**
- **Settings**
- Logs (this could help the users find it more easily and it's also important for the support tickets on Discord)
- **Hide Game**
- **Uninstall**
- **Update** (it can be placed at the bottom as it just appears when the game has a new update). 

Another suggestion it's to make this component not be limited to the card size, but this will need to tweak the code and overcomplicate for now. But I'm ok with designing it if we feel we need to add more options here.

**Comment by iamdavid0091** [comment]
**The menu will include the following options:**
- Logs
- Hide game
- Uninstall


Note that Logs points the user to "System Logs".
